layout.css

```diff
@@ -0,0 +1,95 @@
+@media(min-width:768px){
+  body {
+    margin: auto; 
+   width: 960px;
+   height: 768px;
+ background-color: bisque;
+ }
```

Comment:
The indentation for the above section is inconsistent. For ease of code review, and for your own debugging, keep it consistent. A linter can help with this.